scripts/run_utils.sh

```diff
@@ -134,7 +134,7 @@ create_tmux_windows() {
   while [[ $idx -lt $NODES ]]
   do
     # Start a new tmux window with the next node. Give it a name to add some more spice
-    tmux new-window -n 'the dude abides again...' "cd node_$idx && ./artemis --config=./config/runConfig.$idx.toml --logging=INFO"
+    tmux new-window -n 'the dude abides again...' "cd node_$idx && ./artemis --config=./config/runConfig.$idx.toml --logging=$LOG"
```

Comment:
Why is logging set to log here instead of info?

Reply:
it's a variable.....